frame/wormhole/backing/s2s/src/lib.rs

```diff
@@ -463,10 +448,9 @@ pub mod pallet {
 			if *lane != T::MessageLaneId::get() {
 				return 0;
 			}
-			let mut weight = 0 as Weight;
 			for nonce in messages.begin..=messages.end {
 				let result = messages.message_dispatch_result(nonce);
-				let message_id = nonce_to_message_id(lane, nonce);
+				let message_id: BridgeMessageId = (*lane, nonce);
 				let (user, amount) = <TransactionInfos<T>>::take(message_id);
```

Comment:
```suggestion
				let (user, amount) = <TransactionInfos<T>>::take((*lane, nonce));
```
No need `let message_id: BridgeMessageId = (*lane, nonce);`.